lib/orangelight/voyager_patron_client.rb

```diff
@@ -0,0 +1,206 @@
+require 'faraday'
+require 'faraday-cookie_jar'
+
+class VoyagerPatronClient
+
+  def initialize(patron)
+    @barcode = patron['barcode']
+    @last_name = patron['last_name']
+    @patron_id = patron['patron_id']
+    @ub_id = "1@PRINCETONDB20050302104001" #patron['ub_id'] is this global for all patrons?
```

Comment:
put this in the requests.yml? How important is this parameter anyway? I think I've put random values in it before and still gotten the desired response.